hooks/image/54-ek-launchers

```diff
@@ -0,0 +1,8 @@
+# Expose Endless Key launchers in the app grid (once a specific content pack has been downloaded)
+
+cat <<- 'EOF' > "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"
+	#!/bin/bash
+
+	XDG_DATA_DIRS="/var/lib/endless-key/data/content/xdg/share:${XDG_DATA_DIRS:-/usr/local/share:/usr/share}"
+	echo "XDG_DATA_DIRS=$XDG_DATA_DIRS"
+EOF
```

Comment:
I think the generated script needs to be made executable or systemd won't actually execute it at runtime.
```suggestion
EOF
chmod 755 "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"
```